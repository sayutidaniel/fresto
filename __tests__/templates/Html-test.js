import React from 'react';
import renderer from 'react-test-renderer';
import Html from '../../src/templates/Html';

describe('HTML template', () => {
  it('should render document structure correctly', () => {
    const tree = renderer.create(
      <Html
        scripts={[
          'common.js',
          'app.js',
        ]}
        styles={[
          'common.css',
          'app.css',
        ]}
        title="Fresto"
      >
        Content goes here
      </Html>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
