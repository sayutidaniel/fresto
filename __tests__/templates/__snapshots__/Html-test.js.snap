exports[`HTML template should render document structure correctly 1`] = `
<html
  lang="en">
  <head>
    <title>
      Fresto
    </title>
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport" />
    <link
      href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700"
      rel="stylesheet" />
    <link
      href="common.css"
      rel="stylesheet"
      type="text/css" />
    <link
      href="app.css"
      rel="stylesheet"
      type="text/css" />
  </head>
  <body>
    <div
      id="app">
      Content goes here
    </div>
    <script
      src="common.js"
      type="text/javascript" />
    <script
      src="app.js"
      type="text/javascript" />
  </body>
</html>
`;
